src/main/java/org/broadinstitute/hellbender/utils/recalibration/covariates/ContextCovariate.java

```diff
@@ -1,20 +1,18 @@
 package org.broadinstitute.hellbender.utils.recalibration.covariates;
 
 import htsjdk.samtools.SAMFileHeader;
+import it.unimi.dsi.fastutil.ints.IntArrayList;
+import it.unimi.dsi.fastutil.ints.IntList;
```

Comment:
I'll quote from my comment on your other branch:

"fastutil is not a small dependency -- it consists of thousands and thousands of classes, and when we added it to GATK3 it slowed down the jar shading process by a noticeable amount. It's also a bit absurd for us to have TWO alternate collections frameworks as dependencies -- our dependencies are bloated enough as-is! I strongly recommend that you look for an alternative in guava, or failing that I think even using int[] would be preferable to introducing a new dependency on fastutil."